model/database/registry.yaml

```diff
@@ -518,6 +518,33 @@ groups:
         stability: experimental
         brief: Cosmos DB sub status code.
         examples: [1000, 1002]
+      - id: db.cosmosdb.consistency_level
+        type:
+          members:
+            - id: strong
+              value: "Strong"
+              stability: experimental
+            - id: bounded_staleness
+              value: "BoundedStaleness"
+              stability: experimental
+            - id: session
+              value: "Session"
+              stability: experimental
+            - id: eventual
+              value: "Eventual"
+              stability: experimental
+            - id: consistent_prefix
+              value: "ConsistentPrefix"
+              stability: experimental
+        stability: experimental
+        brief: Account or request consistency level.
+        examples: [Eventual, ConsistentPrefix, BoundedStaleness, Strong or Session]
+      - id: db.cosmosdb.machine_id
```

Comment:
I believe we discussed it in the past - otel provides a bunch of different identifiers for machine - https://github.com/open-telemetry/semantic-conventions/tree/main/docs/resource including `service.instance.id`, `host.id`, `host.name`, k8s identifiers, etc.

It's enabled by users (or auto-detected by OTel distros) and is NOT a client instrumentation concern. 
If client instrumentations would report it, they would all report different things resulting in poor usability. 

I do not support adding this attribute.

Reply:
I agree.